packages/@aws-cdk/aws-events-targets/README.md

```diff
@@ -23,6 +23,7 @@ Currently supported are:
 * Queue a Batch job
 * Make an AWS API call
 * Put a record to a Kinesis stream
+* Put a record to a Kinesis Firehose stream
```

Comment:
```suggestion
* Put a record to a Kinesis Data Firehose stream
```

Reply:
I'd like to align this with [documentation](https://aws.amazon.com/kinesis/data-firehose/?kinesis-blogs.sort-by=item.additionalFields.createdDate&kinesis-blogs.sort-order=desc)